src/osdc/Objecter.h

```diff
@@ -446,9 +446,10 @@ struct ObjectOperation {
 	    for (list<watch_item_t>::iterator i = resp.entries.begin() ;
 		 i != resp.entries.end() ; ++i) {
 	      obj_watch_t ow;
-	      ostringstream sa;
-	      sa << i->addr;
-	      strncpy(ow.addr, sa.str().c_str(), 256);
+              ostringstream ss;
+              ss << i->addr;
+	      strncpy(ow.addr, ss.str().c_str(), 256);
+              ow.addr[255] = '\0'; // make coverity happy
```

Comment:
if fixing the coverity issue helps us to improve the code, i am happy to do so. but this change does not make sense in the sense of correctness or readability or feature.